src/LibraryUtilities.ts

```diff
@@ -573,9 +573,9 @@ export function buildLibraryItemsFromName(typeListNode: TypeListNode, parentNode
 export function pushKeywords(itemData: ItemData, typeListNode: TypeListNode) {
     let keywords = typeListNode.keywords.split(",");
     keywords.forEach(keyword => {
-        itemData.keywords.push(keyword.toLowerCase().trim())
+        itemData.keywords.push(keyword.toLowerCase().replace(/ /g, ''));
```

Comment:
on second thought it looks ok because keywords won't have other kinds of whitespace characters